src/js/components/favorite/Favorite.jsx

```diff
@@ -58,7 +52,7 @@ export class Favorite extends Component {
 }
 
 Favorite.propTypes = {
-    darkTheme: PropTypes.bool,
     checked: PropTypes.bool,
+    className: PropTypes.oneOf(['', 'dark-yellow', 'dark-white']),
```

Comment:
I'm not at all fond of building in themes to components with colors as their names. It's unclear to me why any of these are using themes anyway and we're not just switching some higher-level css file?